writeup/gz_hubble_data.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[twocolumn]{aastex}
+\documentclass[twocolumn]{aastex6}
```

Comment:
I believe I did that and it had some other weird issues... but maybe I was just afraid to. Will try.